config/software/libsodium.rb

```diff
@@ -22,11 +22,18 @@
 license "ISC"
 license_file "LICENSE"
 
-dependency "autoconf"
-dependency "automake"
-dependency "libtool"
+# Depend on the msys2/mingw environment given to us and don't build our
+# own build tools on windows.
+unless windows?
+  dependency "autoconf"
+  dependency "automake"
+  dependency "libtool"
```

Comment:
Oh right, that's what happened with that.  It had been a while and I forgot what was blocking this.

Yeah I never heard back about the broken push-jobs-client build so...  *shrug* I don't mind keeping this around for a bit longer until one of us gets a spare cycle to sort that out.

The only reason I remembered this was because I was poking around in pj client build for the S3 cache changes on Monday.  For windows is going to continue to be red - so you can ignore it.